windows/src/engine/keyman32/appint/aiTIP.cpp

```diff
@@ -154,10 +162,10 @@ extern "C" __declspec(dllexport) BOOL WINAPI TIPProcessKey(WPARAM wParam, LPARAM
     switch(wParam) {
     case VK_CAPITAL:
       if(!isUp) ProcessToggleChange((UINT) wParam);   // I4793
-      KeyCapsLockPress(isUp);   // I4548
+      if(!Updateable) KeyCapsLockPress(isUp);   // I4548
```

Comment:
We only want to process the Caps Lock key event once -- in the first pass (!Updateable).